cmd/kubectl-gadget/deploy.go

```diff
@@ -88,6 +88,27 @@ metadata:
   name: gadget
   namespace: gadget
 ---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: gadget-role
+rules:
+- apiGroups: [""]
+  resources: ["namespaces", "nodes"]
+  verbs: ["get", "watch", "list"]
+- apiGroups: [""]
+  resources: ["pods"]
+  # update is needed by traceloop gadget.
+  verbs: ["get", "watch", "list", "update"]
```

Comment:
Since traceloop only modifies pods in the gadget namespace, could we make it a `Role` and a `RoldBinding` that would only apply to the `gadget` namespace?

Reply:
I can try, this should be good to reduce `update` scope.